assets/icons/fitview.svg

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 30">
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 30" fill="#d8d8d8">
```

Comment:
This color was used in the abstract for the original SVGs. 

The rest of the colors I made up by eyeing it.